src/main/resources/locales/en-US.yml

```diff
@@ -1,155 +1,415 @@
-      remove-stars-user: "Remove Stars"
-    descriptions:
-      selected: "&d Selected"
-
-      view-mode: "&f Allows to choose | &f which top should | &f be displayed."
-      mode: "Shows top ten | islands by [type]."
-      top-value:
-        likes: "&6 [rank]. island by likes | &6 with [value] likes."
-        dislikes: "&6 [rank]. island by dislikes | &6 with [value] dislikes."
-        rank: "&6 [rank]. island by rank | &6 with [value] rank points."
-        stars: "&6 [rank]. island by stars | &6 with [value] stars."
-      likes-values: "Island has: | [likes] likes."
-      likes-dislikes-values: "Island has: | [likes] likes | [dislikes] dislikes | [rank] rank points."
-      stars-values: "Island has: | [stars] stars | [votes] votes."
-      unknown: "&c unknown player"
+      # Buttons in Likes Manage GUI (Player GUI where he can add/remove likes)
+      add_like:
+        name: "&f&l Like"
+        description: |-
+          &7 Allows to add or remove
+          &7 a like for this island.
+        cost: "&7 It will cost &b&o $[number]"
+      add_dislike:
+        name: "&f&l Dislike"
+        description: |-
+          &7 Allows to add or remove
+          &7 a dislike for this island.
+        cost: "&7 It will cost &b&o $[number]"
+      add_star:
+        name: "&f&l [number] Star"
+        description: |-
+          &7 Allows to add or remove
+          &7 a star for this island.
+        cost: "&7 It will cost &b&o $[number]"
+      # Buttons in Likes View Panel
+      like:
+        name: "&f&l Likes Count"
+        description: |-
+          &7 Shows number of likes
+        likes: "&o&b [number] likes"
+      like_rank:
+        name: "&f&l Place by Likes"
+        description: |-
+          &7 Shows island place
+          &7 in top by likes
+        rank: "&o&b [number]. place"
+        not-ranked: "&o&b Not in the top."
+      dislike:
+        name: "&f&l Dislikes Count"
+        description: |-
+          &7 Shows number of dislikes
+        dislikes: "&o&b [number] dislikes"
+      dislike_rank:
+        name: "&f&l Place by Dislikes"
+        description: |-
+          &7 Shows island place
+          &7 in top by dislikes
+        rank: "&o&b [number]. place"
+        not-ranked: "&o&b Not in the top."
+      overall:
+        name: "&f&l Rank"
+        description: |-
+          &7 Shows rank value
+        value: "&o&b [number] rank"
+      overall_rank:
+        name: "&f&l Place by Rank"
+        description: |-
+          &7 Shows island place
+          &7 in top by rank
+        rank: "&o&b [number]. place"
+        not-ranked: "&o&b Not in the top."
+      stars:
+        name: "&f&l Average Stars"
+        description: |-
+          &7 Shows number of average stars
+        stars: "&o&b [number] stars"
+      stars_rank:
+        name: "&f&l Place by Stars"
+        description: |-
+          &7 Shows island place
+          &7 in top by stars
+        rank: "&o&b [number]. place"
```

Comment:
Same comment. I assume it would appear as, for example, "5. place", but that doesn't make sense.

In English, we use 1st place, 2nd place, 3rd place, etc. and the first three ranks are irregular. You could write a little method to convert numbers to numerical places, like this:

1st
2nd
3rd
4th
5th
6th
7th
8th
9th
10th
11th
12th
13th
...
21st
22nd

Or just remove the word "place".

Reply:
How about:
```
    rank: "&o&b Place: [number]"
```